src/settings/LocationLocations/LocationManager.js

```diff
@@ -398,28 +406,29 @@ class LocationManager extends React.Component {
       libraryId,
     } = this.state;
 
-    const campuses = [];
-    const libraries = [];
-
-    const institutions = ((resources.institutions || {}).records || []).map(i => (
-      { value: i.id, label: `${i.name}${i.code ? ` (${i.code})` : ''}` }
-    ));
+    const institutions = get(resources.institutions, 'records', [])
```

Comment:
As far as I understand, the main idea of the fix is replace null by empty strings and put ids to the child component.
Actually the code looks better after changes, but I'm not sure that the refactoring should be provided in scope of this issue because it takes some additional time to check changes.

Reply:
Thanks, I decided to clean up stuff a bit.